Cargo.toml

```diff
@@ -269,6 +269,7 @@ fuse = { path = "vm/devices/support/fs/fuse" }
 lx = { path = "vm/devices/support/fs/lx" }
 lxutil = { path = "vm/devices/support/fs/lxutil" }
 plan9 = { path = "vm/devices/support/fs/plan9" }
+lower_vtl_permissions_guard = { path = "vm/devices/support/lower_vtl_permissions_guard" }
```

Comment:
nit: sort

Reply:
is it sorted based on the name of the crate, or the path of where it is?

Reply:
Oh good point, I just saw the l and p right above it.